_layouts/training_experience_list.html

```diff
@@ -0,0 +1,110 @@
+---
+layout: base
+---
+
+{% assign url_parts = page.url | split:'/' %}
+{% assign topic_name = url_parts[2] %}
+{% assign tutorial_name = url_parts[4] %}
+
+{% assign topic = site.data[topic_name] %}
+{% assign contributors = site.data['contributors'] %}
+{% assign instances = site.data['instances'] %}
+{% assign topic_material = site.pages | topic_filter:topic_name %}
+{% assign tutorial = topic_material | where:"tutorial_name",tutorial_name %}
+
+<header>
+    <nav class="navbar navbar-expand-lg navbar-dark">
+        <div class="container">
+            <a class="navbar-brand" href="{{ site.baseurl }}/">
+                <img src="{{ site.baseurl }}/{{ site.small_logo }}" height="30" alt="Galaxy Training Network logo">
+                {{ site.title }}
+            </a>
+
+            <button class="navbar-toggler navbar-toggler-right" type="button" data-toggle="collapse" data-target="#top-navbar" aria-controls="top-navbar" aria-expanded="false" aria-label="Toggle navigation">
+                <span class="navbar-toggler-icon"></span>
+            </button>
+            <div class="collapse navbar-collapse" id="top-navbar">
+                <ul class="navbar-nav">
+                    <li class="nav-item">
+                        <a class="nav-link" href="{{ site.baseurl }}/topics/{{ topic.name }}" title="Go back to list of tutorials">
+                            {% icon topic %} {{ topic.title }}
+                        </a>
+                    </li>
+
+                    {% include _includes/help.html %}
+
+                    <li class="nav-item">
+                        <a class="nav-link" href="{{ site.github_repository }}/edit/master/topics/{{ topic.name }}/tutorials/{{ page.tutorial_name }}/tutorial.md">
+                            {% icon github %} Edit
+                        </a>
+                    </li>
+                </ul>
+            </div>
+        </div>
+    </nav>
+</header>
+
+<div class="container main-content">
+    <section class="tutorial">
+        <h1>{{ page.title }}</h1>
```

Comment:
There was no title displayed for me locally. Maybe we could have a default title there